_data/simple-icons.json

```diff
@@ -5295,6 +5295,21 @@
             "hex": "512DA8",
             "source": "https://favro.com/login"
         },
+        {
+            "title": "FCC",
+            "hex": "1C3664",
+            "source": "https://www.fcc.gov/logos",
+            "guidelines": "https://apps.fcc.gov/oetcf/kdb/forms/FTSSearchResultPage.cfm?id=27980&switch=P",
```

Comment:
Not really sure if this counts as a guidelines link 🤔

Reply:
It is. It's the labeling guideline for the FCC label. We'd better to link this for users.